### DISCUSSION TITLE
Issue with Updating Pwsh to 7.3.2 [Solved]

### DISCUSSION THREAD

**Discussion by Xelbayria** [ScoopInstaller/Main#4408] [Q&A] [answered]
Hello, 

I was updating `pwsh` to the latest version as I normally would do but I got a error message `ERROR The following instances of "pwsh" are still running. Close them and try again.`. I tried everything to try and shut it down. It still won't shut down. I had to rebooted my computer and the result is the same. I do not know what's going on. I was using `CMD` to update `pwsh`. 

One thing I find is quite strange is that `CMD.exe` was using the exactly same color as `pwsh` is using to highlight the text. Do anyone know how to fix it? Thanks for the helping!

**Comment by Xelbayria** [comment, marked as answer]
nvm. I found the answer to my own question. I haven't tried `Windows Powershell` to update `pwsh` and it works. Please ignore this topic.

**Comment by darthwalsh** [comment]
For more context, the thread https://github.com/ScoopInstaller/Main/issues/3572 has a lot of discussion about this :)

**Reply by Xelbayria** [reply]
👍
